e2e/vm/additional_disk_test.go

```diff
@@ -22,7 +22,6 @@ const (
 var testAdditionalDisk = func(o *option.Option) {
 	ginkgo.Describe("Additional disk", ginkgo.Serial, func() {
 		ginkgo.It("Retains container user data after the VM is deleted", func() {
-			command.Run(o, "volume", "create", volumeName)
```

Comment:
Is this removed because the [`run` command below](https://github.com/runfinch/finch/pull/521/files#diff-387dfcdc65084af4019ef8892c8316ab9a32370a66da4c01438ba1be56818079R29) already creates the volume?

Reply:
😓 good catch unintentionally removed